src/python/dxpy/bindings/dxproject.py

```diff
@@ -296,6 +296,8 @@ def new(self, name, summary=None, description=None, protected=None,
         :type protected: boolean
         :param restricted: If provided, whether the project should be restricted
         :type restricted: boolean
+        :param download_restricted: If provided, whether the project should be download restricted
```

Comment:
Consider rewording this as: "If provided, whether external downloads should be restricted"